dotCMS/gradle.properties

```diff
@@ -1,4 +1,4 @@
-dotcmsReleaseVersion=master
+dotcmsReleaseVersion=issue-20295-filenames-with-blank-spaces-cause-problems
```

Comment:
same here

Reply:
I wish there was an easier way to force a PR to run against a specific version of enterprise without having to make all these changes. Then see how people think you "probably" added them by accident and request you get them removed. 
Yes. they need to be removed.